From b4393a13b654263194ceb6fad0db9cd6e1821e39 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Danis?= <frederic.danis@collabora.com>
Date: Wed, 24 Apr 2024 15:38:40 +0200
Subject: [PATCH BlueZ 2/2] plugins: wake-policy: Only allow Peripherals to
 wake up the host
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This prevents Headsets/Earbuds to wake up the host.
---
 plugins/wake-policy.conf | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/plugins/wake-policy.conf b/plugins/wake-policy.conf
index e6220bad1..35e006f3d 100644
--- a/plugins/wake-policy.conf
+++ b/plugins/wake-policy.conf
@@ -9,13 +9,13 @@
 # Possible values for each Major Class:
 # true,false,<Minor Class allowed list (integer separated by ;)>
 
-#Computer=true
-#Phone=true
-#LAN/Network=true
-#Audio/Video=true
+Computer=false
+Phone=false
+LAN/Network=false
+Audio/Video=false
 #Peripheral=true
-#Imaging=true
-#Wearable=true
-#Toy=true
-#Health=true
-#Uncategorized=true
+Imaging=false
+Wearable=false
+Toy=false
+Health=false
+Uncategorized=false
-- 
2.34.1

